.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+name: CI
+
+permissions: read-all
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  check-scripts:
+    name: Check scripts
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./scripts
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@main
+      - name: Install the latest version of uv
+        uses: eifinger/setup-uv@v1
```

Not sure, see https://docs.astral.sh/uv/guides/integration/github/

```suggestion
        uses: eifinger/setup-uv@v5
```

---

Oh thanks, not sure where I copied the `v1` from 🙈